README.md

```diff
@@ -1,10 +1,11 @@
 # Hanoi Towers [![hanoi-tower](https://snapcraft.io/hanoi-tower/badge.svg)](https://snapcraft.io/hanoi-tower)
 
-# ![Hanoi Towers Logo](https://i.ibb.co/23T2nF8/Hanoi-5.png)
+# ![Hanoi Towers Logo](hanoitowers/res/HanoiTowers_Banner_Web.png)
+
 
 ***************************
-## What is Hanoi Towers
-The Hanoi Tower is one of the most popular puzzles of the 19th century. Three bars are given, on one of which eight rings are strung, the rings differ in size and smaller. The problem is to transfer the pyramid from the eight rings for the least number of moves to another rod. At a time, only one ring is allowed to carry, and you can not put a larger ring on less
+## About  Hanoi Towers
+This version of the game is a test project of QasarApp, which means that it is distributed free of charge and without ads. This game allows you not only to "kill time", it is useful, corrupting logic. The game is rather a simulator for the development of logical thinking, you can spend time playing at the same time doing self-development. The rules of the game are quite simple, you will move smaller objects to larger ones in such a way that you get a tower of blocks going up from larger to smaller, try to make as few moves as possible, this is the part of the game that develops logic, think over the moves to the front. In our implementation, the level of complexity is gradually increasing. Initially, you start with a small pyramid, after you solve this problem you move on to the next level. At each subsequent level, one new block is added, the level numbers correspond to the number of pyramid blocks, you yourself can switch between the levels that you have completed. The game also has a competitive component, you can register an online account to get into the table of records and also learn about the achievements of other players.
```

Comment:
Break the entire description into paragraphs with your own headings. nobody reads a solid text to the end